src/main/java/shef/servlets/NewLiveStreamServlet.java

```diff
@@ -32,17 +32,22 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Since this method is putting a new livestream into Datastore, isn't doPost() more appropriate?

Reply:
Agreed, I think it should be doPost. In fact, doesn't your create-live-stream.html send a post request to /new-live-stream, not a get? Does this even work then?

Reply:
It does work! Also, sorry, probably should've clarified in the description bc it is a bit weird:

This is a doGet because - if you look at script.js - I'm storing livestreams by calling fetch. So, this only works if the servlet IS a doGet. I poked around because I was wondering if the doPost --> create entity and doGet --> retrieve entities is a convention, but it turns out that the only doPost/doGet difference is that "You should use doGet() when you want to intercept on HTTP GET requests. You should use doPost() when you want to intercept on HTTP POST requests. That's all. Do not port the one to the other or vice versa (such as in Netbeans' unfortunate auto-generated processRequest() method). This makes no utter sense." a la https://stackoverflow.com/questions/2349633/doget-and-dopost-in-servlets.

Reply:
I did remove the form bit (form method=POST), though, because that was residual. It does nothing.